numpile.py

```diff
@@ -9,18 +9,25 @@
 import pprint
 import string
 import numpy as np
-from itertools import tee, izip
 
 from textwrap import dedent
 from collections import deque, defaultdict
 
-import llvm.core as lc
-import llvm.passes as lp
-import llvm.ee as le
-from llvm.core import Module, Builder, Function, Type, Constant
+import llvmlite.llvmpy.core as lc
```

Comment:
If switching to llvmlite, I would personally just switch to its native API. (Well, as a full disclosure, it may be just me. My aim would be to port numpile to MicroPython+[ullvm](http://github.com/pfalcon/ullvm/), and while providing a llvmlite-like wrapper API for ullvm would be useful, I definitely won't add implementation for old unmaintained API of llvmpy).